lighthouse-core/test/computed/speedline-test.js

```diff
@@ -52,14 +53,12 @@ describe('Speedline gatherer', () => {
 
   it('does not change order of events in traces', () => {
     // Use fresh trace in case it has been altered by other require()s.
-    const pwaJson = fs.readFileSync(__dirname +
-        '/../fixtures/traces/progressive-app.json', 'utf8');
-    const pwaTrace = JSON.parse(pwaJson);
+    const pwaTrace = readJson('lighthouse-core/test/fixtures/traces/progressive-app.json');
```

Comment:
move to top-level imports?

Reply:
See comment L55. This is avoiding mutation.